go_modules/lib/dependabot/go_modules/path_converter.rb

```diff
@@ -1,19 +1,32 @@
-# typed: true
+# typed: strict
 # frozen_string_literal: true
 
+require "sorbet-runtime"
+
 require "dependabot/go_modules/native_helpers"
 
 module Dependabot
   module GoModules
     module PathConverter
+      extend T::Sig
+
+      sig do
+        params(path: T.untyped)
```

Comment:
Please educate me: why is this untyped and not a string?

Reply:
Good catch! It should be.